bson/src/main/org/bson/json/JsonReader.java

```diff
@@ -250,6 +251,9 @@ public BsonType readBsonType() {
                     currentValue = visitUUIDConstructor();
                 } else if ("new".equals(value)) {
                     visitNew();
+                } else if ("?".equals(value)) {
```

Comment:
This part was easy, since we already support raw strings dues to the shell syntax